tests/pw/.env

```
@@ -0,0 +1,52 @@
+# Database Configuration (Docker wp-env)
+DB_HOST_NAME=localhost
+DB_USER_NAME=root
+DB_USER_PASSWORD=password
+DATABASE=tests-wordpress
+DB_PORT=9998
+DB_PREFIX=wp
+
+# REST API - MUST use query string format for Docker!
+SERVER_URL=http://localhost:9999/?rest_route=
+
+SERVER_URL=http://localhost:9999/wp-json
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The SERVER_URL key is duplicated

(DuplicatedKey)